tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/balo/object/ObjectInBaloTest.java

```diff
@@ -589,4 +591,10 @@ public void testObjectReferingNonAbstractObjLoadedFromBalo() {
 //                                          "non-public fields or methods",
 //                                  42, 6);
 //    }
+
+    @AfterClass
+    public void tearDown() {
+        BaloCreator.clearPackageFromRepository("test-src/balo/test_projects/test_project", "testorg", "foo");
```

Comment:
Since we use `org.ballerinalang.test.BCompileUtil#compileAndCacheBalo` to set up the balo now, I don't think calling this will make a difference.